integration_tests/sdk/setup_integration.py

```diff
@@ -170,7 +176,9 @@ def setup_data_integrations(filter_to: Optional[str] = None) -> None:
 
         # Setup the data in each of these integrations.
         integration = client.integration(integration_name)
-        if integration.type() == ServiceType.SNOWFLAKE:
+        if integration.type() == ServiceType.REDSHIFT:
```

Comment:
Maybe we can make a dictionary with key = integration.type() and value = function (_setup_{integration}_data)

Reply:
It might be the same readability and we'll have to check if the integration exists in the map, so I might leave this as is for now.